DS_1/LinkedListRotation.java

```diff
@@ -0,0 +1,235 @@
+package datastructure_1;
+import java.util.Scanner;
+public class LinkedListRotation {
+	/**
+	 * @return the head node of linked list after creating the linked list
+	 */
+	public Node createList() 
```

Comment:
No need to make method public, if the method is to be used within this file only.